src/test/java/WebCrawlerTest.java

```diff
@@ -114,4 +114,30 @@ public void shouldGetItemsQueue() {
         //assert
         assertEquals(expected, actual);
     }
+
+
+    @Test
+    public void checkRootURLsAmount() {
+        //arrange
+        WebCrawler webCrawler = new WebCrawler();
+        //act
+        webCrawler.readUrl("https://localhost/tci", "a", "abs:href");
+        //assert
+        assertEquals(11,webCrawler.linksList.size());
+    }
+
+    @Test
+    public void addDifferentItemsToItemList() {
+        //arrange
+        WebCrawler webCrawler = new WebCrawler();
+        Book bb = mock(Book.class);
```

Comment:
Choose better naming conventions, it's not clear. Just name it book or mock_book